tempto-core/src/main/java/io/prestosql/tempto/query/JdbcQueryExecutor.java

```diff
@@ -94,13 +96,16 @@ public Connection getConnection()
     private QueryResult execute(String sql, QueryParam... params)
             throws QueryExecutionException
     {
+        requireNonNull(sql, "sql is null");
+        requireNonNull(params, "params is null");
+
         if (connection == null) {
             openConnection();
         }
 
         sql = removeTrailingSemicolon(sql);
 
-        LOGGER.debug("executing on {} query {} with params {}", jdbcUrl, sql, params);
+        LOGGER.debug("executing on {} query [{}] with params: {}", jdbcUrl, sql, asList(params));
```

Comment:
why aren't we using `%s` here?

LOGGER -> log?

Reply:
> why aren't we using `%s` here?

tempto uses slf4j

> LOGGER -> log?

sure, next time